code/mapping/README.md

```diff
@@ -0,0 +1,52 @@
+# Intermediate layer
+
+The intermediate layer is a 2D top-down map for sensor fusion and entity tracking. The map can be visualized with the visualization node in the mapping_visualization package
+
```

Comment:
We should add a hint here that this packages gets compiled by cython with the hack in cmake. 
Nobody reads more than the first 10 Lines but in my opinion it is very important for the understanding.

Reply:
I agree